tests/parens/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -599,11 +603,11 @@ $var = $var += 10;
 
 $var = 10 + 20 + 30;
 $var = 10 + 20 + 30;
-$var = 10 + 20 + 30;
+$var = 10 + (20 + 30);
```

Comment:
From a mathematical perspective, the braces could be removed here because `+` is associative. Is prettier for JS implementing that?

Reply:
Mathematically it is associative, often in programming languages with floating point numbers it is not (see for example this example I posted about JavaScript https://github.com/prettier/prettier/issues/187#issuecomment-385807181). I still find it really weird to have them there and I did a maths degree :)

Reply:
Nevermind, turns out floating point addition is not always associative (see [here](https://github.com/prettier/prettier/issues/187#issuecomment-385807181)).